### ISSUE TITLE
[question] Allow omitting channel from a package name?

### ISSUE THREAD

**Issue by db4** [conan-io/conan#9515]
Since using channels to promote packages is discouraged in favour of repo to repo move I would like to get rid of channels for my private packages. But I would like to retain user field to differ them from everything else. Something like `package/1.0.0@user`. Looks like it's not possible now - one either has to use `package/1.0.0` or `package/1.0.0@user/channel`. Am I right? If yes, maybe supporting package names with user but without channel makes sense?

**Comment by memsharded** [comment]
Yes, this will make sense for conan 2.0, I think it was discussed that it will be allowed to drop only the channel.

**Comment by memsharded** [comment]
Packages with only user and no channel are now possible in the released 2.0-alpha.4, both creation and search work:
```
(conan2_36) λ conan list recipes *
Local Cache:
  pkg
    pkg/0.1@myuser
```
